build.gradle

```diff
@@ -208,7 +209,11 @@ dependencies {
     provided "appeng:RotaryCraft:${config.rotc.version}:api"
     provided ("appeng:appliedenergistics2:${config.ae2.version}:dev") {
         exclude module: 'buildcraft'
-    }
+    }*/
+
```

Comment:
@28Smiles Thanks for updating the integration! What's the status here, i.e. do you think you can port the bits that currently require non-api classes to use the API?